src/components/publishing/display/display_panel.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react"
+import styled, { StyledFunction } from "styled-components"
+import Colors from "../../../assets/colors"
+import { crop, resize } from "../../../utils/resizer"
+import Fonts from "../fonts"
+
+interface DisplayPanelProps extends React.HTMLProps<HTMLDivElement> {
+  unit: any
+  campaign: any
+}
+interface DivUrlProps extends React.HTMLProps<HTMLDivElement> {
+  imageUrl: string
+  hoverImageUrl: string
+}
+
+const DisplayPanel: React.SFC<DisplayPanelProps> = props => {
+  const { unit, campaign } = props
+  return (
+    <DisplayPanelContainer
+      imageUrl={crop(unit.image_url, { width: 680, height: 284 })}
+      hoverImageUrl={resize(unit.logo, { width: 680 })}
+    >
+      <Image />
```

Comment:
To return to the link idea, would assume that this Image is meant to link out too